src/xdocs/config_coding.xml

```diff
@@ -3431,6 +3431,86 @@ for (String line: lines) {
       </subsection>
     </section>
 
+    <section name="NoArrayTrailingComma">
+      <p>Since Checkstyle 8.28</p>
+       <subsection name="Description" id="NoArrayTrailingComma_Description">
+         <p>
+           Checks that array initialization do not contain a trailing comma.
+         </p>
```

Comment:
There should be a short "rationale" section.
For example:

> Rationale: JLS allows trailing commas in arrays and enumerations, but does not allow them in other locations. To unify the coding style, the use of trailing commas should be prohibited.

Reply:
Agree

Reply:
Done